content/zh/docs/concepts/signals/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 信号
+description: 了解 OpenTelemetry 支持的遥测类别
+aliases:
+  - /docs/concepts/data-sources
+  - /docs/concepts/otel-concepts
+weight: 11
+default_lang_commit: a0ac78ec4ec9c360ea8f86481cf01bf2b82140f1
+---
+
+OpenTelemetry 的目的是收集、处理和导出**[信号][]**。
```

Comment:
```suggestion
OpenTelemetry 的目的是收集、处理和导出 **[信号][]** 。
```
The [preview](https://deploy-preview-5521--opentelemetry.netlify.app/zh/docs/concepts/signals/) looks not good.